canonical/ubuntu.com
# [Sunbeam GA Launch] - /openstack revamp

## Done

- Updated page as per [doc](https://docs.google.com/document/d/1eqHK0hc9v7Mn-XV36QRyNkJenTXgBQR_-RULKm5rZ5o/edit)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: https://ubuntu-com-12935.demos.haus/openstack
    - Be sure to test on mobile, tablet and desktop screen sizes
- Make sure copy changes are correct
- Check "Get your OpenStack" functionality ("I will manage OpenStack" should reveal 2 cards in step 2)
- Check carousel (logos needed from visual, will update once they are finalized)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-3628
